### DISCUSSION TITLE
How do I disable the spell check inside the quill text area?

### DISCUSSION THREAD

**Discussion by Rohan-cp** [slab/quill#4453] [Q&A]
<img width="733" alt="Screenshot 2024-10-20 at 06 23 17" src="https://github.com/user-attachments/assets/901019c1-fb36-460c-a93b-0bd92a448965">

I want to remove this behaviour where 'mexican' is underlined

**Comment by VahidN** [comment]
Add `spellcheck="false"` attribute to the outer DIV.
